lib/src/main/java/uk/gov/di/ipv/core/library/helpers/VcHelper.java

```diff
@@ -46,23 +53,37 @@ public static boolean isSuccessfulVc(
                         evidenceArray.toJSONString(),
                         new TypeToken<List<CredentialEvidenceItem>>() {}.getType());
 
-        return isValidEvidence(credentialEvidenceList, isFraudAllowedA01);
+        return isValidEvidence(credentialEvidenceList, ciScoresMap, ciScoreThreshold);
+    }
+
+    public static int calculateCiScore(
+            List<String> ciList, Map<String, ContraIndicatorScores> ciScoresMap) {
+        Set<String> ciSet = ciList.stream().collect(Collectors.toSet());
+        int ciScore = 0;
+        for (String ci : ciSet) {
+            ContraIndicatorScores scoresConfig = ciScoresMap.get(ci);
+            ciScore += scoresConfig.getDetectedScore();
+        }
+        return ciScore;
     }
 
     private static boolean isValidEvidence(
-            List<CredentialEvidenceItem> credentialEvidenceList, boolean isFraudAllowedA01) {
+            List<CredentialEvidenceItem> credentialEvidenceList,
+            Map<String, ContraIndicatorScores> ciScoresMap,
+            int ciScoreThreshold) {
         try {
             for (CredentialEvidenceItem item : credentialEvidenceList) {
                 if (item.getType().equals(CredentialEvidenceItem.EvidenceType.EVIDENCE)) {
-                    return Gpg45EvidenceValidator.isSuccessful(item);
+                    return Gpg45EvidenceValidator.isSuccessful(item, ciScoresMap, ciScoreThreshold);
```

Comment:
I think for all these successful checks we can remove any checking of CI's. I don't think it really determines if a VC is successfull anymore, as if the CI is mitigated then it is successful.

I think these "success" checks should just be based on the actual gpg45 score value, i.e if it is 0 or not.

Reply:
I think the reality is that apart from Fraud, VCs won't have a non zero profile score if they contain a CI. Lets just leave this for now.